zadaci/sa-kolokvijuma/2020/RA/T5/G8/resenje.c

```diff
@@ -0,0 +1,182 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define MAX_REZERVISAO 26
+#define MAX_NAZIV_BALETA 9
+
+typedef struct balet {
+    unsigned sediste;
+    char nazivBaleta[MAX_NAZIV_BALETA];
+    char rezervisao[MAX_REZERVISAO];
+    unsigned cena;
+    struct balet *next;
+}BALET;
+
+void init_list(BALET **head) {
+    *head = NULL;
+}
+
+char *lower(char str[])
+{
+    int i;
+
+    for(i=0; i<strlen(str); i++)
+    {
```

Comment:
Please be consistent with your coding style. Everywhere else this bracket is at the end of the line above.